src/app/[locale]/_utils/__tests__/jwt.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { JWTUser } from '../../_model/JWTUser';
+import { User } from '../../_model/User';
+import { extractToken, parseJwt, userFromJwtToken } from '../jwt';
+
+describe('Utilities', () => {
+  describe('extractToken', () => {
+    test('extracts token from URL', () => {

```

Comment:
Should we be more expressive in those descriptions?

Reply:
Sure